Clustering/Hdbscan/src/test/java/org/tribuo/clustering/hdbscan/TestHdbscan.java

```diff
@@ -0,0 +1,228 @@
+
+        assertArrayEquals(expectedLabelPredictions, actualLabelPredictions);
+        assertArrayEquals(expectedOutlierScorePredictions, actualOutlierScorePredictions);
+    }
+
+    @Test
+    public void testBasicTrainPredict() {
+        runBasicTrainPredict(t);
```

Comment:
Can you add `Helpers.testModelSerialization(model,ClusterID.class);` to one of these tests. The model serialization helper checks the provenance and model serialization and we want that check on all model implementations.